apps/marsha/templates/services/app/cronjob_check_harvested.yml.j2

```diff
@@ -0,0 +1,69 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  labels:
+    app: marsha
+    service: app
+    version: "{{ marsha_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "check-harvested-{{ deployment_stamp }}"
+  namespace: "{{ project_name }}"
+spec:
+  schedule: "0 3 * * *"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 1
+  concurrencyPolicy: Forbid
+  suspend: {{ suspend_cronjob | default(false) }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: "check-harvested-{{ deployment_stamp }}"
+          labels:
+            app: marsha
+            service: app
+            version: "{{ marsha_image_tag }}"
+            deployment_stamp: "{{ deployment_stamp }}"
+        spec:
+{% set image_pull_secret_name = marsha_image_pull_secret_name | default(none) or default_image_pull_secret_name %}
+{% if image_pull_secret_name is not none %}
+          imagePullSecrets:
+            - name: "{{ image_pull_secret_name }}"
+{% endif %}
+          containers:
+            - name: marsha
+              image: "{{ marsha_image_name }}:{{ marsha_image_tag }}"
+              imagePullPolicy: Always
+              command:
+                - "/bin/bash"
+                - "-c"
+                - python manage.py check_harvested
+              env:
+                - name: DJANGO_SETTINGS_MODULE
+                  value: marsha.configs.settings
+                - name: DJANGO_CONFIGURATION
+                  value: "{{ marsha_django_configuration }}"
+                - name: POSTGRES_DB
+                  value: "{{ marsha_postgresql_database }}"
+                - name: POSTGRES_HOST
+                  value: "marsha-{{ marsha_postgresql_host }}-{{ deployment_stamp }}"
+                - name: POSTGRES_PORT
+                  value: "{{ marsha_postgresql_port }}"
+                - name: DJANGO_ALLOWED_HOSTS
+                  value: "{{ marsha_host | blue_green_hosts }}"
+                - name: DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH
+                  value: "{{ marsha_cloudfront_private_key_path }}"
```

Comment:
you don't need these 2 but ok if you want the template to the same on all jobs.

Reply:
If I remember, this setting is mandatory. Running the management command will saying that this setting is missing.

Reply:
ok then :man_shrugging: